lib/web/apiserver_test.go

```diff
@@ -772,6 +772,7 @@ func TestClusterNodesGet(t *testing.T) {
 	nodes := clusterNodesGetResponse{}
 	require.NoError(t, json.Unmarshal(re.Bytes(), &nodes))
 	require.Len(t, nodes.Items, 1)
+	require.Equal(t, nodes.TotalCount, 1)
```

Comment:
Technically, the signature is `require.Equal(t, EXPECTED, ACTUAL)`.

We've switched the expected/actual values, which is fine if the test passes, but can result in a confusing error message if there's a value.